src/database/video_emotion_database.py

```diff
@@ -40,6 +41,15 @@ def find(self, col,
                                   ).sort([('last_updated', -1)]
                                          ).limit(1)[0]
 
+    def find_all_same_id(self, name, song_id):
+        results = []
+        data =  self._db[name].find({'song_id': song_id})
```

Comment:
Should really be "col" instead of "name" as it isn't the db name. It's the name of the collection.